### DISCUSSION TITLE
No density for jp2?

### DISCUSSION THREAD

**Discussion by Jmuccigr** [ImageMagick/ImageMagick#2842] [Help] [answered]
This old thread suggests that there is no way to encode density in jp2 files: https://www.imagemagick.org/discourse-server/viewtopic.php?t=28893

Is that still the case?

This page suggests that exiftool can read, but not write the display resolution: https://exiftool.org/TagNames/Jpeg2000.html#DisplayResolution

When I try, I can have exiftool set the usual Xresolution and Yresolution, but they don't seem to make a difference in the right place: the resolution of the image stays at 72.

**Comment by dlemstra** [comment, marked as answer]
There is still nothing available in the API that allows us to read or write the density of a jp2 image.

**Reply by Jmuccigr** [reply]
Bummer. Thanks for the reply.
